home/service/search.py

```diff
@@ -56,11 +56,17 @@ def _build_entity_types(self, entity_types: list[str]) -> tuple[ResultType, ...]
         )
         return chosen_entities if chosen_entities else default_entities
 
+    def _format_query_value(self, query: str) -> str:
+        query_pattern: str = r"^[\"'].+[\"']$"
+        # We expect a quoted query to be an exact match therefore when not, we replace
```

Comment:
I find this comment confusing out of context 😕 

Can we say something like "datahub treats any query with underscores as exact, so if the string is not quoted, we convert underscores to spaces so that we get partial matches as well"